Utilities/MCStepLogger/src/AnalysisBasic.cxx

```diff
@@ -0,0 +1,281 @@
+#include <unordered_map>
+#include "MCStepLogger/Analysis.h"
+
+using o2::StepInfo;
+using o2::MagCallInfo;
+using namespace o2::mcstepanalysis;
+
+class AnalysisBasic : public Analysis
```

Comment:
Consider renaming `BasicAnalysis` and adding some documentation what kind of analysis this is doing.

Why is this done in a source file? This looks very weird, in particular since you later include this file somewhere else.

Reply:
is split now into header and source. Was a remnant...